doc/lua_api.txt

```diff
@@ -7530,14 +7582,14 @@ Used by `minetest.register_node`.
         drop = "",
         -- Name of dropped item when dug.
         -- Default dropped item is the node itself.
-        -- Using a table allows multiple items, drop chances and tool filtering.
-        -- Tool filtering was undocumented until recently, tool filtering by string
+        -- Using a table allows multiple items, drop chances and item filtering.
+        -- Item filtering was undocumented until recently, item filtering by string
```

Comment:
This isn't fully within the scope of the PR, but would you be willing to do something about the "until recently" in here? I have hangups about seeing documentation dated like this, especially when 'recently' was in fact 2 years ago.